### PR TITLE
refs #6161 - Add test to make sure override is marked on create

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

**Comment by domcleal** [comment]
This has changed the output of another test.  If you can use a factory instead of introducing a fixture, please do, it'll reduce side effects across tests and is our current preference.

**Comment by tbrisker** [comment]
@domcleal alright, changed it to use a factory instead.

File: test/functional/api/v2/override_values_controller_test.rb [hunk 1]
```diff
@@ -20,12 +20,21 @@ class Api::V2::OverrideValuesControllerTest < ActionController::TestCase
     assert_equal 2, override_values["results"].length
   end
 
+  test 'should mark override on creation' do
+    k = FactoryGirl.create(:lookup_key, :puppetclass => puppetclasses(:two))
+    assert_equal false, k.override
```
Comment (domcleal) [inline test/functional/api/v2/override_values_controller_test.rb]: `refute k.override`, and assert below (rather than assert_equal true)

**Comment by GregSutcliffe** [comment]
Tests fine here - reverting 73eb580f causes this new test to fail as expected. Merged as 40f9c48, thanks!

**Pull request closed without merging**